README.md

```diff
@@ -76,9 +78,9 @@ file.
     * The **serverConfig** section contains values related to the Pega Infinity server and SDK Content Server.
       * **infinityRestServerUrl** indicates where the SDK expects to find the Infinity REST server. The default configuration is set to **https://localhost:1080/prweb**. You will need to change this if you are using a
       different path to access your Infinity server.
-      * **appAlias** specifies the application alias the operator that is logging in will use. (E.g., "MediaCo").
+      * **appAlias** is the application alias that operatorIDs logging in to this instance of the React SDK content server will use. (E.g., "MediaCo").
       When specified, this value will be used to further constrain the REST URL
-      that the Constellation JavaScript Engine uses when making REST calls to Infinity.
+      that the Constellation JavaScript Engine uses when making REST calls to Infinity. It is **_strongly recommended_** that the appAlias value be set before putting an application into production to further improve the security of the application.
```

Comment:
...to allow operators who may have the application access group listed as the non-primary access group to also work.  (suggest that be mentioned rather than security)

Reply:
Ok. I've updated.